backend/src/services/accessRequest.ts

```diff
@@ -91,6 +91,12 @@ export async function removeAccessRequest(user: UserInterface, accessRequestId:
 
   await accessRequest.delete()
 
+  try {
+    await removeAccessRequestReview(accessRequestId)
+  } catch (error) {
+    log.warn(error, 'Error when creating Release Review Requests. Approval cannot be given to this Access Request')
```

Comment:
Instead of using `log.error` or `log.warn` we have some utility functions that are used for throwing specific errors. They can be found in [here.](https://github.com/gchq/Bailo/blob/main/backend/src/utils/error.ts)

Here's an example of how to use one:

`if (!myValue) {
  throw BadReq('myValue was undefined!', myValue)
}`